scripts/react_native_pods.rb

```diff
@@ -57,8 +57,7 @@ def use_react_native! (options={})
   end
 end
 
-def add_flipper_pods!
-  version = '~> 0.30.0'
+def add_flipper_pods!(version = '~> 0.30.2')
```

Comment:
I’m going to rename this to `use_flipper!` to be in line with the other methods in this file.